Tests/XMLCoderTests/CombineTests.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CombineTests.swift
+//  XMLCoder
+//
+//  Created by Adam Sharp on 9/29/19.
+//
+
+#if canImport(Combine)
+import Combine
+import Foundation
+import XCTest
+import XMLCoder
+
+private let xml = """
+<?xml version="1.0" encoding="UTF-8"?>
+<foo>
+    <name>Foo</name>
+</foo>
+""".data(using: .utf8)!
+
+private struct Foo: Decodable {
+    var name: String
+}
+
+@available(iOS 13.0, macOS 10.15.0, tvOS 13.0, watchOS 6.0, *)
```

Comment:
I wonder if the solution would be to remove `macOS 10.15.0` clause until that's released? We would avoid crashing on CI until then too. After all I wouldn't mind tagging a minor release just for enabling that later when Catalina is released and available on CI.

Reply:
Removing macOS from the availability annotation would instead result in a compiler error. The issue is that we successfully compile & link, but then attempt to run the tests on a platform where the framework doesn't exist and instead crash when the symbol can't be loaded.

If we want to exclude the test from macOS altogether, we could add `#if !os(macOS)` to the compiler directive at the top of the file. The tests won't be exercised on CI then, so it might still make sense to add a CI job that runs on Xcode 11 + iOS?

Reply:
The Xcode 11 job already tests on macOS, iOS and tvOS, so that should be fine.